core/templates/pages/exploration-editor-page/editor-tab/graph-directives/exploration-graph.component.html

```diff
@@ -43,7 +43,7 @@
       </div>
     </mat-card>
   </div>
-  <div class="oppia-exp-graph-modal-icon-container" (click)="openStateGraphModal()">
+  <div class="oppia-exp-graph-modal-icon-container e2e-test-oppia-mobile-graph-resize-button" (click)="openStateGraphModal()">
```

Comment:
In other class names we don't use 'oppia' so maybe we should omit it here too.

Reply:
dfd

Reply:
Done.